spec/tree_spec.rb

```diff
@@ -1,14 +1,63 @@
 require 'rspec'
 require 'tree'
 
+tree = Tree.new
+
 describe 'Tree' do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(Tree_class.is_a? 'Class').to be_true
+  end
+
+  it 'grows one foot' do
```

Comment:
why might grouping your tests into contexts be a good idea?
www.betterspecs.org/#contexts